api/README.md

```diff
@@ -753,3 +754,74 @@ empty string, which will yield all test results for the selected runs.
 ```
 
 </details>
+
+## Metadata results
+
+### /api/metadata
+
+This endpoint accepts POST and GET requests.
+
+- GET request returns Metadata Link Information by product, and requires product parameters;
+
+- POST request searches Metadata Link by link url, and requires product parameters and payload.
+
+__Parameters__
+
+__`product`__ : browser[version[os[version]]]. e.g. `chrome-63.0-linux`
+
+#### Link Query
+  link: [pattern]
```

Comment:
For consistency of style:
```suggestion
__`link`__: [pattern]
```

Also remove the spaces at the beginning of next line.